changelog

```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.14.3] - 2023-09-20
+
+### Added
+- default value (mdule = 0) for `zdc` constraints
+
```

Comment:
In [semantic versioning](https://semver.org/) changing anything but _Fixed_ is not a patch, so go for 2.15.0

Reply:
typo: `mdule` -> `module`

Reply:
And the links table at the bottom is waiting for an update too :) https://github.com/cms-l1-globaltrigger/tm-vhdlproducer/blob/e5f25516a891caa50e5ddeaa4f2e08bc913890cd/changelog#L285-L288

Reply:
Done!